### PR TITLE
Reworked dbpedia scripts

### PR DESCRIPTION
**Pull request by TimHenkelmann**
Had to change the following things:
- added "done ...." to process.send message such that it can be checked in cli.js
- add checks for "body" return value of request, otherwise cheerio throws error
- fixed dateOfBirth and dateOfDeath ( needs to be $(this) in the each loop)
- auto-formating

### CODE REVIEW

File: dbpedia/dbPedia_Composers.js [hunk 1]
```diff
@@ -20,10 +21,36 @@ function replaceURLAndUnderscore(str) {
     return replaceAll(str, "_", " ")
 }
 
+
+url.forEach(function (value) {
+    get_URL(value);
+});
+
+
+function get_URL(value, callback) {
```
Comment (kordianbruck) [inline dbpedia/dbPedia_Composers.js]: So WTF are you doing? CamelCase? UnderScores? :dancing_men:

File: dbpedia/dbPedia_Composers.js [hunk 2]
```diff
@@ -82,7 +109,7 @@ function getResults(queryURL, callback) {
             //psuedonym
             var pseudonym = [];
             if ($('span[property="dbp:psuedonym"]').text() && $('span[property="dbp:psuedonym"]').text() != "") {
-                $('span[property="dbp:psuedonym"]').each(function(index) {
+                $('span[property="dbp:psuedonym"]').each(function (index) {
```
Comment (kordianbruck) [inline dbpedia/dbPedia_Composers.js]: When using a complex selector it pays off to save the object to a local var

File: dbpedia/dbPedia_Composers.js [hunk 3]
```diff
@@ -123,14 +150,14 @@ function getResults(queryURL, callback) {
...
             var release = [];
             if ($('a[rel="dbp:artist"]').attr('href') && $('a[rel="dbp:artist"]').attr('href') != "") {
-                $('a[rel="dbp:artist"]').each(function(index) {
+                $('a[rel="dbp:artist"]').each(function (index) {
                     var rel = replaceURLAndUnderscore($(this).attr('href'));
                     release.push(rel);
                 });
             }
 
             if ($('a[rel="dbo:artist"]').attr('href') && $('a[rel="dbo:artist"]').attr('href') != "") {
```
Comment (kordianbruck) [inline dbpedia/dbPedia_Composers.js]: Especially if you use the same var like 20 times

File: dbpedia/dbpedia_Classical_musicians_by_century.js [hunk 4]
```diff
@@ -31,180 +30,183 @@ function replaceURLAndUnderscore(str) {
 }
 
 function fetch1(link1) {
```
Comment (kordianbruck) [inline dbpedia/dbpedia_Classical_musicians_by_century.js]: So what does fetch1 do? Naming this function a bit better would give other people an insight into what is supposed to happen here

**Review by kordianbruck — COMMENTED** [review]
I stopped looking at some point as this PR is quite big and seems to have a lot of code duplication

File: dbpedia/dbPedia_Composers.js (hunk 3 above)
Comment (TimHenkelmann) [reply dbpedia/dbPedia_Composers.js]: Every selector is used **at most** 3 times, but you're still right. I will adapt all dbpedia scripts.

File: dbpedia/dbpedia_Classical_musicians_by_century.js (hunk 4 above)
Comment (TimHenkelmann) [reply dbpedia/dbpedia_Classical_musicians_by_century.js]: not my code but will incorporate the changes

File: dbpedia/dbPedia_Composers.js (hunk 1 above)
Comment (TimHenkelmann) [reply dbpedia/dbPedia_Composers.js]: not my code but will incorporate the changes

**Comment by TimHenkelmann** [comment]
@ShilpaGhanashyamGore @angelinrashmi2000 @LukasNavickas Done with my fixes, can you review this please?

**Review by kordianbruck — APPROVED** [review]
Nice rework! Excellent work <3

**Pull request merged**